### ISSUE TITLE
Owl API needs an architecture rebuff

### ISSUE THREAD

**Issue by phi-line** [FoothillCSClub/OwlAPI#47]
While working on #46 I realized that we need to move away from all the api scripts in root. This caused some difficult while trying to get Travis setup to run automatically.

Specifically the modules found in root should be moved to a folder named `OwlAPI`.
With an `__init__.py` in that folder, it should be easier to call OwlAPI modules without the need of an `__init__.py` in the root.

This issue should be tackled after the merge of @TryExceptElse PR

**Comment by phi-line** [comment]
This is an example project schema:
```
.
├── src/
│   ├── db/
│   ├── server.py
│   ├── data_scraper.py
│   ├── __init__.py
├── tests/
│   ├── test_server.py
│   ├── generate_test_data.py
│   ├── __init__.py
├── frontend/
│   ├── static/
│   ├── templates/
├── examples/
│   ├── single/
│   ├── batch/
├── .gitignore
├── Pipfile
├── Pipfile.lock
└── README.md
```

Files to be deprecated: `crossdomain.py` `settings.py`

**Comment by TryExceptElse** [comment]
The thing is that we really only have one file that the server code resides in; server.py. (and a tiny amount in settings.py) The scripts would end up in a separate folder entirely, if we were following standard python practice. Are you sure we can't run travis on a single module project?

**Comment by phi-line** [comment]
Fixed by #51 and #53
